config/routes.rb

```diff
@@ -6,5 +8,6 @@
   get "up" => "rails/health#show", as: :rails_health_check
```

Comment:
If this route doesn't be part of your project, please remove it.

Reply:
Fixed!